### ISSUE TITLE
Support Multiple Library Imports in Jenksinfile

### ISSUE THREAD

**Issue by codezninja** [nvuillam/npm-groovy-lint#246]
So [Jenkinsfile recommends importing multiple shared libraries](https://www.jenkins.io/doc/book/pipeline/shared-libraries/#loading-libraries-dynamically) like so

```groovy
/* Accessing multiple libraries with one statement */
@Library(['my-shared-library', 'otherlib@abc1234']) _
```

npm-groovy-lint is by default failing for the `_` on 
* The variable [_] in class None is not used  UnusedVariable
* Variable named _ in class None does not match the pattern [a-z][a-zA-Z0-9]*  VariableName

Current workaround is to add `// groovylint-disable-line UnusedVariable, VariableName`. Could we by default add support for multiple library support?

**Comment by nvuillam** [comment]
As errors are returned by CodeNarc, rules UnusedVariable and VariableName should be updated there, you can post an issue on https://github.com/CodeNarc/CodeNarc

**Comment by codezninja** [comment]
@nvuillam CodeNarc suggest disabling the rule in our ruleset. Which I amd already. So I'm guessing anyone that uses this npm-groovy-lint on jenkinsfile that have library imports will all have to disable that rule for that line.

Would it make sense to document that for other users? I could open a PR for that.

**Comment by nvuillam** [comment]
> @nvuillam CodeNarc suggest disabling the rule in our ruleset. Which I amd already. So I'm guessing anyone that uses this npm-groovy-lint on jenkinsfile that have library imports will all have to disable that rule for that line.
> 
> Would it make sense to document that for other users? I could open a PR for that.

@codezninja your PR to update documentation will be very welcome :)
